graph2tac/tfgnn/predict.py

```diff
@@ -423,7 +811,9 @@ def ranked_predictions(self,
         if available_global is not None:
             raise NotImplementedError('available_global is not supported yet')
 
-        inference_output = self._inference_model(state, allowed_model_tactics)
+        best_log_probs, best_tactics = self._inference_model(state, allowed_model_tactics)
+        return (best_tactics.numpy()[0], np.exp(best_log_probs.numpy()[0]))
+
         predict_output = PredictOutput(state=None, predictions=[])
 
         # go over the tactic_expand_bound batches
```

Comment:
TODO: Remove all the code below this.  (And probably lots of other code. :smile:)